src/models/insight.ts

```diff
@@ -0,0 +1,13 @@
+import { Information } from './information';
+
+export interface Insight<T> {
+	value: T;
+	confidence: number;
+	evidence: Array<Information>;
+}
+
+export interface BusinessInsights {
+	capacity?: Insight<number>;
```

Comment:
This field might not be immediately feasible / can be moved to the walk stage. Even with the results of the third party models we have only been able to consistently determine the existence of the 2-3 front most chairs for each photo
```suggestion
```

Reply:
This interface is no longer part of this PR